warp10/src/main/java/io/warp10/script/MemoryWarpScriptStack.java

```diff
@@ -1172,9 +1205,9 @@ public void exec(Macro macro) throws WarpScriptException {
           }
         }
         if (null == name) {
-          throw new WarpScriptException("Exception" + (i < n ? (" at '" + statementString + "'") : "") + " in section '" + section + "'", ee);
+          throw new WarpScriptException("Exception" + (i < n ? (" at '" + statementString + "'") : "") + " in section '" + section + "'" + (getAttribute(WarpScriptStack.ATTRIBUTE_LAST_STMTPOS) instanceof String ? (" at position " + getAttribute(WarpScriptStack.ATTRIBUTE_LAST_STMTPOS)) : ""), ee);
         } else {
-          throw new WarpScriptException("Exception" + (i < n ? (" at '" + statementString + "'") : "") + " in section '" + section + "' called from macro '" + name + "'", ee);
+          throw new WarpScriptException("Exception" + (i < n ? (" at '" + statementString + "'") : "") + " in section '" + section + "'" + (getAttribute(WarpScriptStack.ATTRIBUTE_LAST_STMTPOS) instanceof String ? (" at position " + getAttribute(WarpScriptStack.ATTRIBUTE_LAST_STMTPOS)) : "") + " called from macro '" + name + "'", ee);
```

Comment:
Otherwise, this leads to :
In TRY, it nearly works as expected with LINEON, but not without LINEON:
```
LINEON T STMTPOS 44
<% 
  -
%> 
<% %> 
<%   ERROR <% 'message' GET  %> F LMAP   %> TRY
```
- ` "Exception at '-' in section '[Line #3]' at position 3:0:6"` , line is correct, position is not correct)
- If your remove LINEON in the previous example, there is no position at all